src/UnitTests/LanguageServerClient/UnitTests.LanguageServerClient/UnitTests.LanguageServerClient.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>NU1701</NoWarn>
```

Comment:
Special disabled warning about Microsoft.VisualStudio.LanguageServer.Client being built on a different framework. but just using interfaces from Microsoft.VisualStudio.LanguageServer.Client so it should be an issue.